plugins/sqlfluff-templater-dbt/src/dbt_templater/templater.py

```diff
@@ -1,379 +1,378 @@
+
+            self.dbt_selector_method = PathSelector(self.dbt_manifest)
+        else:
+            from dbt.graph.selector_methods import (
+                MethodManager as DbtSelectorMethodManager,
+                MethodName as DbtMethodName,
+            )
+
+            selector_methods_manager = DbtSelectorMethodManager(
+                self.dbt_manifest, previous_state=None
+            )
+            self.dbt_selector_method = selector_methods_manager.get_method(
+                DbtMethodName.Path, method_arguments=[]
+            )
+
+        if self.formatter:
+            self.formatter.dispatch_compilation_header(
+                "dbt templater", "Project Compiled."
+            )
+
+        return self.dbt_selector_method
+
+    def _get_profiles_dir(self):
+        """Get the dbt profiles directory from the configuration.
+
+        The default is `~/.dbt` in 0.17 but we use the
+        PROFILES_DIR variable from the dbt library to
+        support a change of default in the future, as well
+        as to support the same overwriting mechanism as
+        dbt (currently an environment variable).
+        """
+        from dbt.config.profile import PROFILES_DIR
+
+        dbt_profiles_dir = os.path.abspath(
+            os.path.expanduser(
+                self.sqlfluff_config.get_section(
+                    (self.templater_selector, self.name, "profiles_dir")
+                )
+                or PROFILES_DIR
+            )
+        )
+
+        if not os.path.exists(dbt_profiles_dir):
+            templater_logger.error(
+                f"dbt_profiles_dir: {dbt_profiles_dir} could not be accessed. Check it exists."
+            )
+
+        return dbt_profiles_dir
+
+    def _get_project_dir(self):
+        """Get the dbt project directory from the configuration.
+
+        Defaults to the working directory.
+        """
+        dbt_project_dir = os.path.abspath(
+            os.path.expanduser(
+                self.sqlfluff_config.get_section(
+                    (self.templater_selector, self.name, "project_dir")
+                )
+                or os.getcwd()
+            )
+        )
+        if not os.path.exists(dbt_project_dir):
+            templater_logger.error(
+                f"dbt_project_dir: {dbt_project_dir} could not be accessed. Check it exists."
+            )
+
+        return dbt_project_dir
+
+    def _get_profile(self):
+        """Get a dbt profile name from the configuration."""
+        return self.sqlfluff_config.get_section(
+            (self.templater_selector, self.name, "profile")
+        )
+
+    @staticmethod
+    def _check_dbt_installed():
+        try:
+            import dbt  # noqa: F401
+        except ModuleNotFoundError as e:
+            raise ModuleNotFoundError(
+                "Module dbt was not found while trying to use dbt templating, "
+                "please install dbt dependencies through `pip install sqlfluff[dbt]`"

```

Comment:
Is this error message still accurate? (The part about `sqlfluff[dbt]`?)

Reply:
I've just removed this now - if we're assuming that dbt is now installed because it's a requirement - then this will never be called.